Make the role playable by each host

MaxVigne opened the pull request:
Change the behavior of the role to be run for each host to add (and not run once globally, with one host doing all the work). 

This makes the role code simpler with better variable handling and makes parallelism available.

Includes #14

Commit:
Make the role playable by each host

Change the behavior of the role to be run for each host and not run
once globally, with one host doing all the work.

Commit:
Update README

pilou- committed:
Use inventory variables instead of add_host parameters

pilou- committed:
Remove boot_docker_network add boot_docker_networks

Before this change boot_docker_network was always required.
Handle multiple networks.

pilou- committed:
update documentation

pilou- committed:
user should be able to choose Docker server

pilou- committed:
Don't duplicate networks

pilou- commented on tasks/main.yml:
```diff
@@ -1,18 +1,46 @@
+    purge_networks: "{{ boot_docker_purge_networks }}"
+    # These are non privileged containers running systemctl, see:
+    # https://developers.redhat.com/blog/2016/09/13/running-systemd-in-a-non-privileged-container/
+    security_opts: seccomp=unconfined  # required on Debian
```
The fedora seccomp profile could be used here too.

pilou- committed:
Use a Systemd compatible seccomp profile ...

when security options are avaible. For example, this isn't the case in Travis
environment:
> linux seccomp: seccomp profiles are not supported on this daemon,
> you cannot specify a custom seccomp profile

pilou- committed:
Check that seccomp profile is used
